internal/git/provider/repo.go

```diff
@@ -1,7 +1,17 @@
 package provider
 
+import "fmt"
+
 type Repository struct {
 	Project string
 	Org     string
 	Name    string
 }
+
+func (r Repository) String() string {
+	if r.Project == "" {
+		return fmt.Sprintf("%s/%s", r.Org, r.Name)
```

Comment:
Is this meant to support Azure DevOps repositories in the future?

Reply:
Yes